src/ui/map.js

```diff
@@ -197,7 +197,7 @@ const defaultOptions = {
  * @example
  * var map = new mapboxgl.Map({
  *   container: 'map',
- *   center: [-122.420679, 37.772537],
+ *   center: [-122.4679, 37.772537],
```

Comment:
Is there any reason to change the coordinates of the example?

Reply:
Accidental change 😂